src/Event/ReadModel/JSONLD/EventLDProjector.php

```diff
@@ -386,15 +384,14 @@ private function getEventType(\stdClass $eventJsonLD): ?EventType
         $eventType = null;
         foreach ($eventJsonLD->terms as $term) {
             if ($term->domain === 'eventtype') {
-                $typeId = $term->id;
                 // This is a workaround to allow copies of events that
                 // have a placeType instead of an eventType.
                 // These events could also be cleaned up in the future
                 // @see https://jira.uitdatabank.be/browse/III-3926
                 try {
-                    $eventType = $this->eventTypeResolver->byId($typeId);
```

Comment:
While there is nothing wrong with this, I would prefer to keep this variable as `$typeId`, because Types, Themes & facilities are not the same(thought they are all terms).